packages/ember-runtime/tests/core/is_array_test.js

```diff
@@ -1,10 +1,10 @@
 import Ember from 'ember-metal/core';
-import { isArray } from 'ember-runtime/utils';
 import ArrayProxy from 'ember-runtime/system/array_proxy';
 
 QUnit.module('Ember Type Checking');
 
 var global = this;
+var isArray = Ember.isArray;
```

Comment:
@rwjblue maybe im reading this incorrectly, if the bug was `Ember.isArray` was wrong (from the public interface), maybe we should actually test that the public interface is correct ?